server/src/util/sanitizer.go

```diff
@@ -0,0 +1,32 @@
+package util
+import (
+	"fmt"
+	"strings"
+	"regexp"
+)
+
+// Sanitized the input prefix recieved.
+func SanitizePrefix(prefix string) string {
+	// regex expression
+	nonSpecialCharacterRegex := regexp.MustCompile(`[^a-zA-Z/-]+`)
```

Comment:
Aren we sanitzing digits too? I think plan was to keep them?

Reply:
yes, so any not starting with alphabets and "-" we are replacing it with "".